.gitconfig

```diff
@@ -0,0 +1,5 @@
+[merge]
```

Comment:
This file is personal, should be located at `~/` on your computer

Reply:
resolved 4b90437